neuro_flow/expr.py

```diff
@@ -624,6 +710,12 @@ def a(value: str) -> Parser:
 
 EXPR.define(BIN_OP_EXPR | UNARY_OP_EXPR | ATOM_EXPR)
 
+LIST_MAKER.define((LSQB + EXPR + many(COMMA + EXPR) + RSQB) >> make_list)
+
+DICT_MAKER.define(
+    (LBRACE + EXPR + COLON + EXPR + many(COMMA + EXPR + COLON + EXPR) + RBRACE)
```

Comment:
I am not sure, but if it is easy to add it would be great to support trailing comma in `DICT_MAKER` and in `LIST_MAKER`. This way it will be closer to python.

Reply:
Sure!